libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -406,10 +406,13 @@ protected TokenEndpointResponse performRequest(LoginOptions options) {
         @Override
         protected void onPostExecute(TokenEndpointResponse tr) {
             if (tr != null && tr.authToken != null) {
-                loginOptions.jwt = tr.authToken;
+                loginOptions.setJwt(tr.authToken);
                 doLoadPage(true);
             }
-            loginOptions.jwt = null;
+            else {
```

Comment:
This is to fix a scenario where if the jwt expired, the app will stuck at a blank screen